### PR TITLE
14.0 mig pos repove pos category rebased

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

**Review by angelmoya — APPROVED** [review]
:+1: 

Code review and functional OK

File: pos_remove_pos_category/static/src/js/models.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/******************************************************************************
+ * Copyright (C) 2015-2016 Akretion (<http://www.akretion.com>).
+ * Copyright (C) 2017-TODAY Camptocamp SA (<http://www.camptocamp.com>).
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ ******************************************************************************/
+
+odoo.define("pos_remove_pos_category.models", function (require) {
+    "use strict";
+
+    var pos_models = require("point_of_sale.models");
```
Comment (ivantodorovich) [inline pos_remove_pos_category/static/src/js/models.js:10]:
```suggestion
    const models = require("point_of_sale.models");
```
Not blocking

File: pos_remove_pos_category/static/src/js/models.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            for (var i = 0 ; i < this.models.length; i++){
+                if (this.models[i].model == "pos.category") {
+                    this.models[i].model = "product.category";
+                    this.models[i].domain = [["available_in_pos", "=", true]];
+                }
+            }
+            return _pos_super.initialize.apply(this, arguments);
+        }
+    });
+});
```
Comment (ivantodorovich) [inline pos_remove_pos_category/static/src/js/models.js]:
```suggestion
    return models;
});

```
or `pos_models` if previous change wasn't done.

This is to play nice with other modules.

**Review by ivantodorovich — APPROVED** [review]
Thanks!
Technically seems ok

Minor not blocking comments.

Can you squash your migration commits into a single `[MIG] pos_remove_pos_category: Migration to 14.0` commit, though?

**Pull request closed without merging**